components/Breadcrumb/src/breadcrumb.css

```diff
@@ -0,0 +1,124 @@
+.denhaag-breadcrumb {
+  --denhaag-breadcrumb-background-color: transparent;
```

Comment:
This line has no influence on the style in my tests. I think we can remove it?

Reply:
The dot's  overrule this css property, by default it's transparent. Without it Safari bugged at my macbook. Rendered "null" in the value box of the editor.